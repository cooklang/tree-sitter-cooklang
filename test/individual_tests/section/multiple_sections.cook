= Section 1 =
First step.

== Section 2 ==
Second step.