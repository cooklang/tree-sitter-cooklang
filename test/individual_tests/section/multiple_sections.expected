(recipe 
  (section
    header: (section_header name: )
    content: (step))
  (section
    header: (section_header name: )
    content: (step)))